src/library/scala/App.scala

```diff
@@ -36,6 +38,7 @@ import scala.collection.mutable.ListBuffer
  *
  *  Future versions of this trait will no longer extend `DelayedInit`.
```

Comment:
for Hacktoberfest, change to `would no longer` [if there were future versions]. Make the PR title: Fix English grammar.

Reply:
you're on the wrong PR for these reviews 😬 this is just a draft so I can work on reflect before the others get through

Reply:
all the code you're commenting on is in #9235

Reply:
Your reply will only serve to loosen my lips.